markdown-preview-eww.el

```diff
@@ -42,19 +42,75 @@
   "Filename of converted html.")
 
 (defvar markdown-preview-eww-waiting-idling-second 1
-  "Seconds of convert waiting")
+  "Seconds of convert waiting.")
 
-(defun markdown-preview-eww-convert-command (output-file-name)
-  (format "require \"redcarpet\"
+(defcustom markdown-preview-eww-convert-command
+  nil
+  "Symbol of markdown convert function.")
```

Comment:
If this is a function name, the variable name and `defcustom` should be as follows (note the default). You should `defgroup` a customization group and put all `defcustoms` under it to make the user experience better.

Even nicer to the user is if you give them a choice between the known conversion functions and an arbitrary function.

```
(defcustom  markdown-preview-eww-convert-function 'markdown-preview-eww-convert-command-redcarpet
  "*A function that is called to convert the Markdown document to HTML that eww can render"
  :group 'markdown-preview-eww
  :type 'function)
```